### PR TITLE
Include sonarqube scanner and improve code coverage

### PR DESCRIPTION
**Pull request by loqo71la**

### CODE REVIEW

**Comment by loqo71la** [comment]
@marcocdlv @JimmyRojas @ziur please review this merge

**Comment by JimmyRojas** [comment]
no observations.. :)

File: apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/AssertMethodCheck.java [hunk 1]
```diff
@@ -72,6 +69,11 @@
     public static final String CHECK_KEY = "A1008";
 
     /**
+     * Stores the index of the second argument.
+     */
+    public static final int TWO = 2;
```
Comment (marcocdlv) [inline apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/AssertMethodCheck.java]: Improve the constant name

File: sonar-apex-plugin/src/main/java/org/fundacionjala/enforce/sonarqube/apex/ApexSquidSensor.java [hunk 2]
```diff
@@ -85,6 +86,11 @@
     private final ResourcePerspectives resourcePerspectives;
 
     /**
+     * Stores a predicate associated with Apex language.
+     */
+    private final FilePredicate filePredicate;
```
Comment (marcocdlv) [inline sonar-apex-plugin/src/main/java/org/fundacionjala/enforce/sonarqube/apex/ApexSquidSensor.java:91]: Improve variable name

File: sonar-apex-plugin/src/main/java/org/fundacionjala/enforce/sonarqube/apex/ApexSquidSensor.java [hunk 3]
```diff
@@ -122,10 +133,7 @@ public ApexSquidSensor(FileSystem fileSystem, ResourcePerspectives perspectives,
      */
     @Override
     public boolean shouldExecuteOnProject(Project project) {
```
Comment (marcocdlv) [inline sonar-apex-plugin/src/main/java/org/fundacionjala/enforce/sonarqube/apex/ApexSquidSensor.java:135]: Improve the name from this method (Example: public boolean executeProject(Project proejct))

**Comment by marcocdlv** [comment]
Please apply the observations

**Pull request merged**